packages/manager/modules/enterprise-cloud-database/src/service/details/backups/recovery/translations/Messages_fr_FR.json

```diff
@@ -7,8 +7,8 @@
   "enterprise_cloud_database_backups_recovery_success": "Restauration lancée avec succès",
   "enterprise_cloud_database_backups_recovery_error": "Impossible de lancer la restauration : {{ message }}",
   "enterprise_cloud_database_backups_recovery_date_placeholder": "Sélectionnez une date à restaurer",
-  "enterprise_cloud_database_backups_recovery_time_slots_helper_to": " en ",
-  "enterprise_cloud_database_backups_recovery_time_slots_helper": "Vous pouvez choisir les dates de ",
+  "enterprise_cloud_database_backups_recovery_time_slots_helper_to": " au ",
```

Comment:
`Vous pouvez choisir entre les dates du {{startDate}} au {{endDate}}`

Instead :)

Reply:
This is done. Thanks :)